lib/immobilienscout/api/attachment.rb

```diff
@@ -25,6 +35,10 @@ def create_metadata_file(params)
         def add_url(is24_id)
           "#{Immobilienscout::Client.api_url}/restapi/api/offer/v1.0/user/me/realestate/#{is24_id}/attachment"
         end
+
+        def order_url(is24_id)
+          "#{Immobilienscout::Client.api_url}/restapi/api/offer/v1.0/user/me/realestate/#{is24_id}/attachment/attachmentsorder"
```

Comment:
might not be done in this PR but I see this part dublicated quite often: `/restapi/api/offer/v1.0/user/me/realestate/`
also for the other endpoints so introducing a constant might make sense (imagine we would switch to v2 then all of those urls have to be edited)

Reply:
Yes good point, so far we just have the main part of the URL as a constant, but the rest is always repeated too, maybe we can take care of this in a new pull request